AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main.py

```diff
@@ -91,4 +91,9 @@ def test_ShapeCollider_CylinderShapeCollides(self, request, workspace, editor, l
     @revert_physics_config
     def test_C15425929_Undo_Redo(self, request, workspace, editor, launcher_platform):
         from .tests import Physics_UndoRedoWorksOnEntityWithPhysComponents as test_module
-        self._run_test(request, workspace, editor, test_module)
\ No newline at end of file
+        self._run_test(request, workspace, editor, test_module)
+        
+    @revert_physics_config
+    def test_AxisAlignedBoxShape_ConfigurationWorks(self, request, workspace, editor, launcher_platform):
+        from .tests.heightfield import AxisAlignedBoxShape_ConfigurationWorks as test_module
+        self._run_test(request, workspace, editor, test_module)
```

Comment:
The name of this test makes it sound like it's testing the AAB Shape in isolation, but in reality it's testing that modifying the AAB Shape affects the Terrain Physics Collider correctly.  Ideally we'd have a name that describes this better, like "TerrainPhysicsCollider_ChangesSizeWithAxisAlignedBoxShapeChanges" or something.

Also, I think this is technically a terrain test, not a physics test.  It should probably be grouped with the other terrain tests.

Reply:
Will make this change in Stabilization/2110.